### PR TITLE
Chart repository

### PR DESCRIPTION
**Pull request by kfox1111**
This change pushes updated charts to a helm chart repository

Signed-off-by: Kevin Fox <Kevin.Fox@pnnl.gov>

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -3,4 +3,42 @@
...
+    set -e
+
+    mkdir -p tmp
+    pushd tmp > /dev/null
+
+    curl https://raw.githubusercontent.com/helm/helm/master/scripts/get > get_helm.sh
+    chmod 700 get_helm.sh
+    ./get_helm.sh
+
+    git clone https://github.com/ceph/csi-ceph
```
Comment (rootfs) [inline deploy.sh]: csi-charts

File: deploy.sh [hunk 2]
```diff
@@ -3,4 +3,42 @@
...
+    git clone https://github.com/ceph/ceph-csi
```
Comment (rootfs) [inline deploy.sh]: by this time the repo and branch is already here

File: deploy.sh [hunk 3]
```diff
@@ -3,4 +3,42 @@
...
+
+    mkdir -p csi-ceph/docs
+    pushd ceph-csi > /dev/null
+    git checkout -b csi-v1.0
```
Comment (rootfs) [inline deploy.sh]: by this time the repo and branch is already here

File: deploy.sh [hunk 4]
```diff
@@ -3,4 +3,42 @@
...
+    pushd tmp > /dev/null
+
+    curl https://raw.githubusercontent.com/helm/helm/master/scripts/get > get_helm.sh
+    chmod 700 get_helm.sh
+    ./get_helm.sh
+
+    git clone https://github.com/ceph/csi-ceph
+    git clone https://github.com/ceph/ceph-csi
+
+    mkdir -p csi-ceph/docs
```
Comment (rootfs) [inline deploy.sh]: how about one chart per directory? `docs/rbd` for rbd

File: deploy.sh (hunk 1 above)
Comment (kfox1111) [reply deploy.sh]: doh. good catch. fixing. all throughout the script too.

File: deploy.sh (hunk 2 above)
Comment (kfox1111) [reply deploy.sh]: its  down a level. and its master based. I'm checking out a directory specifically for 1.0.0.

File: deploy.sh (hunk 3 above)
Comment (kfox1111) [reply deploy.sh]: as far as I can tell, the main checkout by travis is 0.3.

File: deploy.sh (hunk 4 above)
Comment (kfox1111) [reply deploy.sh]: I don't think it will index correctly then?

**Pull request merged**